src/transformers/utils/import_utils.py

```diff
@@ -575,8 +575,15 @@ def is_protobuf_available():
     return importlib.util.find_spec("google.protobuf") is not None
 
 
-def is_accelerate_available():
-    return importlib.util.find_spec("accelerate") is not None
+def is_accelerate_available(partial_state=False):
```

Comment:
```suggestion
def is_accelerate_available(check_partial_state=False):
```
Not super fan of the name otherwise.